about.html

```diff
@@ -24,17 +24,17 @@ <h1>Sandstorm is on a mission</h1>
 
-<p>In early 2017, Sandstorm Development Group <a href="https://sandstorm.io/news/2017-02-06-sandstorm-returning-to-community-roots">ran out of funding</a> and the team primarily <a href="https://sandstorm.io/news/2017-03-13-joining-cloudflare">joined Cloudflare</a>. Sandstorm development continued with a major rewrite of the HTTP proxy, a refactor of the account model, and the addition of localization support. However, the pace of development slowed, and the Oasis <a href="https://sandstorm.io/news/2018-08-27-discontinuing-free-plan">free plan was discontinued</a> in 2018. Finally, <a href="https://sandstorm.io/news/2019-09-15-shutting-down-oasis">Oasis shut down</a> for paying customers at the end of 2019.</p>
+<p>In early 2017, Sandstorm Development Group <a href="/news/2017-02-06-sandstorm-returning-to-community-roots">ran out of funding</a> and the team primarily <a href="/news/2017-03-13-joining-cloudflare">joined Cloudflare</a>. Sandstorm development continued with a major rewrite of the HTTP proxy, a refactor of the account model, and the addition of localization support. However, the pace of development slowed, and the Oasis <a href="/news/2018-08-27-discontinuing-free-plan">free plan was discontinued</a> in 2018. Finally, <a href="/news/2019-09-15-shutting-down-oasis">Oasis shut down</a> for paying customers at the end of 2019.</p>
 
-<p>In 2020, a group of Sandstorm enthusiasts began a community effort to <a href="https://sandstorm.io/news/2020-02-03-reviving-sandstorm">revive development</a> of Sandstorm. <a href="https://github.com/zenhack">Ian Denhardt</a> did a significant amount of work improving the experience of authorizing network requests from Sandstorm apps, refactoring older code, and making the codebase more maintainable. In 2021, this community group <a href="https://sandstorm.io/news/2021-06-17-sandstorm-community-fundraising">participated in FundOSS</a> and was able to pay a developer to modernize our Etherpad package. As of 2022, Sandstorm Development Group has been completely dissolved, and development of the Sandstorm project has transitioned to a community-run model.</p>
+<p>In 2020, a group of Sandstorm enthusiasts began a community effort to <a href="/news/2020-02-03-reviving-sandstorm">revive development</a> of Sandstorm. <a href="https://github.com/zenhack">Ian Denhardt</a> did a significant amount of work improving the experience of authorizing network requests from Sandstorm apps, refactoring older code, and making the codebase more maintainable. In 2021, this community group <a href="/news/2021-06-17-sandstorm-community-fundraising">participated in FundOSS</a> and was able to pay a developer to modernize our Etherpad package. As of 2022, Sandstorm Development Group has been completely dissolved, and development of the Sandstorm project has transitioned to a community-run model.</p>

```

Comment:
These are particularly noteworthy since this entire block exists only in the .org version anyways... and still used a bunch of .io URLs! No fix here, just acknowledging my failures.